drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,32 +2962,104 @@ func (k *K8s) destroyBackupObjects(
 	return nil
 }
 
-// getExpectedAutopilotSizeForPvc calculates exected size of PVC based on autopilot rules and workload
-func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList, wSize uint64) (uint64, error) {
-	var (
-		err             error
-		expectedPVCSize uint64
-	)
+// GetExpectedAutopilotVolumeExpand calculates expected size of PVC based on autopilot rule and workload
+func (k *K8s) GetExpectedAutopilotVolumeExpand(pvc *v1.PersistentVolumeClaim, apRule apapi.AutopilotRule, wSize uint64) (uint64, int, error) {
+	pvcObjSize := pvc.Spec.Resources.Requests[v1.ResourceStorage]
+	pvcSize, _ := pvcObjSize.AsInt64()
+	expectedPVCSize := uint64(pvcSize)
 
 	driver, err := volume.Get(k.VolDriverName)
 	if err != nil {
-		return 0, err
+		return 0, 0, err
 	}
+	if isAutopilotMatchPvcLabels(apRule, pvc) {
+		return driver.EstimateVolumeExpand(apRule, expectedPVCSize, wSize)
+	}
+	return expectedPVCSize, 0, nil
+}
 
-	pvcObjSize := pvc.Spec.Resources.Requests[v1.ResourceStorage]
-	pvcSize, _ := pvcObjSize.AsInt64()
-	expectedPVCSize = uint64(pvcSize)
+// ValidateAutopilotPVCEvents verifies proper alerts and events on resize completion
+func (k *K8s) ValidateAutopilotPVCEvents(ctx *scheduler.Context, coolDownPeriod int) error {
```

Comment:
don't see this method being invoked anywhere

Reply:
this is a method to be called in pvc tests